packages/paste-core/components/timeline/src/TimelineItemIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box } from "@twilio-paste/box";
+import { IconWrapper } from "@twilio-paste/icons/esm/helpers/IconWrapper";
+import * as React from "react";
+
+import { TimelineContext } from "./TimelineContext";
+import type { TimelineItemIconProps } from "./types";
+
+const TimelineItemIcon = React.forwardRef<HTMLElement, TimelineItemIconProps>(
+  ({ icon: Icon, as, display, element = "TIMELINE_ITEM_ICON", color }, ref) => {
+    const { orientation } = React.useContext(TimelineContext);
+
+    return (
+      <Box element={`${element}_WRAPPER`}>
+        {Icon ? (
+          <Box marginY="space20">
+            {/* @ts-expect-error some error in size*/}
+            <IconWrapper as={as} display={display} element={element} color={color} ref={ref} size="sizeSquare50">
+              {/* @ts-expect-error some error*/}
+              <Icon decorative size="sizeSquare50" />
+            </IconWrapper>
+          </Box>
+        ) : (
+          <Box
+            element={`${element}_DOT`}
+            backgroundColor="colorBackgroundStronger"
+            borderRadius="borderRadiusCircle"
+            width="8px"
+            height="8px"
```

Comment:
can you use the space tokens here
```suggestion
            size="sizeBase20"
```